user/invite-new.md

```diff
@@ -4,6 +4,32 @@ description: How to invite a new user
 
 # Invite user
 
-Explanation
+You can invite a new user to your company:
```

Comment:
You can invite a new user to your company.
An invitation email will be send to this user.

Reply:
Fixed